common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -15,6 +15,7 @@
 
 import com.google.common.geometry.S2CellId;
 import com.google.common.math.DoubleMath;
+import com.sun.org.apache.xpath.internal.operations.Mult;
```

Comment:
What is this `import`? I don't think this is used.

Reply:
This must've been added automatically due to a mistype, removed it.